tests/cow_protocol/ethereum/cow_protocol_ethereum_assert_partial_fill.sql

```diff
@@ -0,0 +1,37 @@
+-- Given a list of trades, when we look at their partial_fill, based on the type of the trade we expect partial_fill to behave differently
+WITH unit_test1 AS
+(
+    SELECT
+        *
+    FROM
+        {{ ref('cow_protocol_ethereum_trades') }}
+    WHERE
+        order_uid = '0xaa1568c867c991bd462bcb6ee5939e4e01144d4e9e29bb74c2fa8d50b3afc92c519b70055af55a007110b4ff99b0ea33071c720a64289604'
+        and partial_fill != true
+),
```

Comment:
Are these the test addresses from our team?

Reply:
Yes. They're order_uid's from Ben's [query](https://dune.com/queries/2321165).